sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -3,13 +3,14 @@ Text Analytics is a cloud-based service that provides advanced natural language
 and includes the main features below:
```

Comment:
Consider updating the description of the service here too. Something like https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/textanalytics/azure-ai-textanalytics/README.md